src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -372,7 +372,7 @@ private DetectProperties() {
         NullableStringProperty.newBuilder("detect.blackduck.signature.scanner.arguments")
             .setInfo("Signature Scanner Arguments", DetectPropertyFromVersion.VERSION_4_2_0)
             .setHelp(
-                "A space-separated list of additional arguments to use when running the Black Duck signature scanner.",
+                "A space-separated list of additional arguments to use when running the Black Duck signature scanner. Key-value pairs are treated as replacements.",
```

Comment:
Can be more descriptive.

Reply:
True. I wonder if @sig-cpotts has any specific suggestions.